site/posts/writing-multiple-netcdf-files-in-parallel-with-xarray-and-dask.ipynb

```diff
@@ -0,0 +1,2084 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A typical computation workflow with xarray consists of:\n",
+    "\n",
+    "1. reading one or multiple netCDF files into an xarray dataset backed by dask using `xr.open_mfdataset()` or `xr.open_dataset(chunks=...)`\n",
+    "2. applying some transformation to the input dataset\n",
+    "3. saving the resulting output to disk in a netCDF file using `xr.to_netcdf()`\n",
+    "\n",
+    "\n",
+    "The last step (3) can easily result into a big netCDF file (>=10GB). As a result, this step can take a very long time to complete (since it is run in serial), and sometimes may hang. So, to avoid these issues one can use one of the lesser-used but helpful xarray capabilities `xr.save_mfdataset()`. <!-- TEASER_END -->This function allows users to write multiple datasets to disk as netCDF files simultaneously. `xr.save_mfdataset()` signature looks like this:\n",
```

Comment:
```suggestion
    "The last step (3) can easily result into a big netCDF file (>=10GB). As a result, this step can take a very long time to complete (since it is run in serial), and sometimes may hang. So, to avoid these issues one can use one of the lesser-used but helpful xarray capabilities `xr.save_mfdataset()`. <!-- TEASER_END -->This function allows users to write multiple datasets to disk as netCDF files simultaneously. The `xr.save_mfdataset()` signature looks like this:\n",
```